pre-commit/action.yml

```diff
@@ -58,6 +58,9 @@ runs:
           rm hcledit.tar.gz 2> /dev/null
         fi
 
+        curl -sSL "$(curl -s https://api.github.com/repos/aquasecurity/tfsec/releases/latest | grep -o -E "https://.+?_linux_amd64.tar.gz")" > tfsec.tar.gz
```

Comment:
Thanks for the PR! Two things:
1. Lets wrap this in an `if` block like we do for hcledit above so that users can opt into installing tfsec
2. Lets use a version instead of the latest and users can choose which version to use (the default can be the current latest version)